README.md

```diff
@@ -1 +1,18 @@
 # ood-pyraf-apptainer
+This repo is based off of [ood-remote-desktop-rosgazebo](https://github.com/Harvard-ATG/ood-remote-desktop-rosgazebo) and is used to create a remote desktop environment with pyraf, iraf and DS9 packages installed via conda.
+
+## Installation
+In order to install the app
+
+Note you will need to be in the head node while building any of the apptainer images.
+Apptainer images should be located `/shared/apptainerImages`
+
+- Activate the spack environment and then use activate the apptainer env.
+```bash
+root@ip:/# . shared/spack/share/spack/setup-env.sh
+root@ip:/# spacktivate apptainer
```

Comment:
Neat, didn't know there was an alias for `spack env activate`.

Reply:
There's a corresponding `despacktivate` too